packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -866,9 +866,15 @@ class EditableText extends StatefulWidget {
        showCursor = showCursor ?? !readOnly;
 
   /// Controls the text being edited.
+  ///
```

Comment:
The comments are not needed, because it is common convention that owner, i.e. class that created an object, is responsible for disposal. 

Can you remove them?